tests/integrations/openai_tracker/test_chat_parameters.py

```diff
@@ -0,0 +1,228 @@
+import uuid
+
+import pytest
+
+from greptimeai import collector
+from . import sync_client
+from ..database.db import truncate_tables, get_trace_data_with_retry
+
+
+@pytest.fixture
+def _truncate_tables():
+    truncate_tables()
+    yield
+
+
+def get_lowercase_letters(letters: str) -> str:
+    return letters.lower()
+
+
+def test_chat_completion_n_gt_1(_truncate_tables):
+    user_id = str(uuid.uuid4())
+    model = "gpt-3.5-turbo"
+    n = 2
+    resp = sync_client.chat.completions.create(
+        n=n,
+        messages=[
+            {
+                "role": "user",
+                "content": "1+1=",
+            }
+        ],
+        model=model,
+        user=user_id,
+        seed=1,
+    )
+
+    assert resp.choices[0].message.content == "2"
+    assert len(resp.choices) == n
```

Comment:
we'd better check if `resp.choices[1]` is ok, and compare it with data in trace table